string consumer
float64[] start
float64[] goal
float64 goal_region_radius
bool smooth_plan
bool set_goal_criterion
plant_locations_msg plant_locations